src/SingleProject/Resizetizer/src/nuget/buildTransitive/Microsoft.Maui.Resizetizer.After.targets

```diff
@@ -568,6 +574,7 @@
     </Target>
 
     <Target Name="ResizetizeImages"
+        Condition="'$(EnableMauiImageProcessing)' != 'false'"
```

Comment:
The only other thought, is we generally document every MSBuild property that is public, such as this doc that is synced with MS docs:

https://github.com/xamarin/xamarin-android/blob/main/Documentation/guides/building-apps/build-process.md

Should we prefix all these values with `_` to make them private, but useable? Or is there a decent place to document them? Like:

https://github.com/dotnet/maui/blob/main/src/Workload/README.md

Reply:
I think I want public, but not sure I am ready to commit.